content/spin/javascript-components.md

```diff
@@ -400,6 +400,6 @@ These are some of the suggested libraries that have been tested and confired to
 
 ## Caveats
 
-- All `spinSdk` related functions and methods can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
+- All `spin` related functions and methods (like `Config`, `Redis`, `Mysql`, `Pg`, `Kv` & `Sqlite`)can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
```

Comment:
```suggestion
- All `spin-sdk` related functions and methods (like `Config`, `Redis`, `Mysql`, `Pg`, `Kv` and `Sqlite`) can be called only inside the `handleRequest` function. This includes the usage of `fetch`. Any attempts to use it outside the function will lead to an error. This is due to Wizer using only Wasmtime to execute the script at build time, which does not include any Spin SDK support.
```

(If you prefer "Spin" to "spin-sdk" at the beginning then capital letter and no monospace - "Spin" is the product, `spin` is the command.  Hashtag brand guidelines that I just made up in my head.)